pkg/docker/config/config.go

```diff
@@ -596,7 +600,7 @@ func modifyJSON(sys *types.SystemContext, editor func(auths *dockerConfigFile) (
 		return "", err
 	}
 
-	auths, err := readJSONFile(path, false)
+	auths, err := readJSONFile(pathV)
```

Comment:
This invites asking what `pathV` stands for…

`path, err := getPathToAuth`, and using `path.path` in the 3 users of that value, seems easier than answering that question.